examples/with-astro/README.md

```diff
@@ -0,0 +1,77 @@
+# Turborepo starter with pnpm
```

Comment:
Many parts of this are not updated, can you correct this? (docs & web are astro, for example)